### DISCUSSION TITLE
Analytics template override

### DISCUSSION THREAD

**Discussion by fnagel** [sulu/sulu#7692] [Q&A] [answered]
I'm trying to change the Matomo templates (as those still use the _piwik_ filenames) but without success. Here's my folder structure: 

![image](https://github.com/user-attachments/assets/0a2d33e7-60e9-4959-9808-d7790f17d4c7)

What am I doing wrong here?

https://docs.sulu.io/en/2.5/cookbook/manage-analytics.html
https://symfony.com/doc/6.4/bundles/override.html#override-templates

**Comment by alexander-schranz** [comment, marked as answer]
It seems like there is no `type` directory if look at the path here: https://github.com/sulu/sulu/blob/2.6/src/Sulu/Bundle/WebsiteBundle/Resources/views/Analytics/matomo/body-open.html.twig 

src/`Sulu`/Bundle/`WebsiteBundle`/Resources/views/`Analytics/matomo/body-open.html.twig`

**Reply by alexander-schranz** [reply]
We may should rework the piwik/matomo implemention for 3.0 if you want to create merge request for uptodate body head it would be awesome.

**Reply by fnagel** [reply]
Here you go: https://github.com/sulu/sulu/pull/7693

Should I create a PR for the 2.x branch too?

**Reply by alexander-schranz** [reply]
On 2.x we can not change the templates as that would be a backwards compatibility break.

**Reply by fnagel** [reply]
Ahh yes, that makes sense.

**Comment by fnagel** [comment]
You are totally right. Removed the subfolder and it works. 

Seems the docs are wrong then: https://docs.sulu.io/en/2.5/cookbook/manage-analytics.html#override-analytics-template

Here's a PR: https://github.com/sulu/sulu-docs/pull/823
